src/components/BlockMenu.tsx

```diff
@@ -470,6 +518,16 @@ class BlockMenu extends React.Component<Props, State> {
               />
             </VisuallyHidden>
           )}
+          {uploadFile && (
+            <VisuallyHidden>
+              <input
+                type="file"
+                ref={this.fileInputRef}
+                onChange={this.handleFilePicked}
+                accept="*"
```

Comment:
Makes sense to make this a prop

Reply:
are you talking about the default accept prop?